models/public/index.md

```diff
@@ -319,6 +319,13 @@ Named entity recognition (NER) is the task of tagging entities in text with thei
 | -------------- | -------------- | ------------------------------------------------------ | -------- | ------- | -------- |
 | vehicle-reid-0001 | PyTorch\* | [vehicle-reid-0001](./vehicle-reid-0001/README.md) | 96.31%/85.15 % | 2.643 | 2.183 |
 
+## Background matting
```

Comment:
please add small explanation what is backgraund matting task

Reply:
Explanation added.